### DISCUSSION TITLE
Notifications count

### DISCUSSION THREAD

**Discussion by husseinalaedin** [mantinedev/mantine#3904] [Feature requests]
in the previous version we were able easy to ge the number of messages that are showing, is that possible in v6?
i cannot find that feature

Thanks

**Comment by jt561** [comment]
[Found solution](https://github.com/orgs/mantinedev/discussions/5192#discussioncomment-7655998) for v7

I also had a v6 solution so let me know if you still need that.

@husseinalaedin
